packages/api-explorer/src/scenes/MethodScene/MethodScene.tsx

```diff
@@ -108,8 +108,14 @@ export const MethodScene: FC<DocMethodProps> = ({ api }) => {
         <Aside width="50rem">
           <ComponentsProvider
             globalStyle={false}
+            loadGoogleFonts
             themeCustomizations={{
-              colors: { background: colors.text, text: colors.background },
+              fontFamilies: { brand: 'Google Sans' },
+              colors: {
+                key: colors.key,
+                background: colors.text,
+                text: colors.background,
+              },
```

Comment:
Using `ExtendComponentsThemeProvider` will allow simplification here as it will absorb previous settings of theme and only change relevant parts specified in the themeCustomizations:

```suggestion
              colors: {
                background: colors.text,
                text: colors.background,
              },
```